addons/event_stand/models/event_stand.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class EventStand(models.Model):
+    _name = 'event.stand'
+    _description = 'Event Stand'
+
+    name = fields.Char('Name', required=True)
+    event_id = fields.Many2one(
+        'event.event', string='Event', required=True)
+    stand_type_id = fields.Many2one(
+        'event.stand.type', string='Booth Type',
+        domain="[('event_id', '=', event_id)]", required=True)
+    can_be_shared = fields.Boolean(related='stand_type_id.can_be_shared')
+    state = fields.Selection([
```

Comment:
reserved does not seem to be used in compute method.

What is its purpose ? Isn't it simply used for a "sold out" ? Could be it be replaced by a boolean ?